IceCubesApp/Resources/Localization/Localizable.xcstrings

```diff
@@ -78625,10 +78639,24 @@
       }
     },
     "Telemetry" : {
-
+      "localizations" : {
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "数据采集"
+          }
+        }
+      }
     },
     "Telemetry by TelemtryDeck" : {
```

Comment:
It's supposed to be TelemetryDeck, I've pushed a fix to main

Reply:
I think there's no longer conflicts now after rebasing.